tests/test_data/callback_data_provider.py

```diff
@@ -110,3 +111,6 @@ def __init__(self):
             shared_secret=self.shared_secret,
             internal_reference=self.internal_reference,
         )
+        self.sample_enrolment_request = NewEnrolmentRequest(
```

Comment:
I think NewEnrolmentRequest is also not needed here

Reply:
Yes, that's right. I'll check the data provider and remove unnecessary stuff.